modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -97,7 +97,7 @@ This procedure creates these tools channels:
 | OS Version | Base Channel                     | Tools Channel
 | {es} 6     | RHEL Expanded Support 6          | RES6-SUSE-Manager-Tools x86_64
 | {es} 7     | RHEL Expanded Support 7          | RES7-SUSE-Manager-Tools x86_64
-| {es} 8     | RHEL or SLES ES or CentOS 8 Base | RES8-Manager-Tools-Pool for x86_64 and RES8-Manager-Tools-Updates for x86_64
+| {es} 8     | RHEL or SLES ES | RES8-Manager-Tools-Pool for x86_64 and RES8-Manager-Tools-Updates for x86_64
```

Comment:
The channel / product name does not change just because CentOS 8 is now unsupported

Reply:
reverted